packages/amplication-client/src/Components/FeatureIndicator.tsx

```diff
@@ -32,9 +32,10 @@ const WarningTooltip = styled(
 
 const CLASS_NAME = "amp-feature-indicator";
 
-export const tooltipDefaultTextUpgrade = "Upgrade";
-
-export const tooltipDefaultText = `Explore this feature, included in your 7-day Enterprise trial. ${tooltipDefaultTextUpgrade} for continued access.`;
+export const defaultTextStart =
+  "Explore this feature, included in your 7-day Enterprise trial.";
+export const defaultTextEnd =
+  "today to ensure continued access and discover additional hidden functionalities!";
```

Comment:
```suggestion
  "Today to ensure continued access and discover additional hidden functionalities!";
```

Reply:
@abrl91, no need, this text coming after the dynamic text: "Talk to us"/ "Talk with our CEO"